tools/render-test/d3d-util.cpp

```diff
@@ -42,8 +42,11 @@ using namespace Slang;
     {
         switch (format)
         {
-            case DXGI_FORMAT_R32_FLOAT:
-            case DXGI_FORMAT_D32_FLOAT:            return DXGI_FORMAT_R32_TYPELESS;
+            case DXGI_FORMAT_R32_FLOAT:     /* fallthru */
```

Comment:
Not important to change, but I agree with your comment on the other PR that when you have:

```c++
case A:
case B:
case C:
    /* code */
```

The fallthrough behavior is much more clear, and probably doesn't need the `/* fallthru */` comment to mark it. For me, the newline before the body code is more than enough to make things clear.